src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/DependencyTreeFlags.cs

```diff
@@ -56,35 +59,20 @@ public static class DependencyTreeFlags
         /// implementations. This is to have a way to distinguish dependency nodes in general.
         /// </summary>
         public static readonly ProjectTreeFlags DependencyFlags
-                = ProjectTreeFlags.Create("Dependency")
-                + ProjectTreeFlags.VirtualFolder
-                + ProjectTreeFlags.BubbleUp
+                = Dependency
+                + ProjectTreeFlags.Reference
```

Comment:
>Reference [](start = 35, length = 9)

Doesn't this contradict the comment above about this flag being used with all nodes?

Reply:
I don't see the contradiction. Can you elaborate?

Note that this flag was being added via `GenericUnresolvedDependencyFlags` and `GenericResolvedDependencyFlags` below. This just moves it higher up.